docker-compose.yml

```diff
@@ -263,6 +263,8 @@ services:
     image: ghcr.io/datadog/dd-apm-test-agent/ddapm-test-agent:v1.37.0
     ports:
       - "127.0.0.1:${DD_TRACE_AGENT_PORT}:9126"
+      - "127.0.0.1:4317:4317"
+      - "127.0.0.1:4318:4318"
```

Comment:
We have to add environment variables to these two ports, since they will be exposed in the developer's host machine, and might conflict with an already running service. (the way to do it similar to `DD_TRACE_AGENT_PORT` above: I'd follow the trail of `DD_TRACE_AGENT_PORT` and do something similar)

Reply:
The testagent supports running otlp severs on different ports. I will use the same configs here: https://github.com/DataDog/dd-apm-test-agent/blob/master/ddapm_test_agent/agent.py#L1868C37-L1868C51